pkgs/development/python-modules/kneaddata/default.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "kneaddata";
+  version = "0.7.7-alpha";
+  pyproject = true;
+
+  dependencies = [ setuptools ];
+
+  src = fetchFromGitHub {
+    owner = "biobakery";
+    repo = "kneaddata";
+    rev = "refs/tags/${version}";
+    hash = "sha256-8pXabwMGNZETEXP0A31SInj37pvogyKpJAaAY7aTyns=";
+  };
+
```

```suggestion

  nativeCheckInputs = [ unittestCheckHook ];

  unittestFlagsArray = [ "kneaddata/tests/ '*.py'" ];

  pythonImportsCheck = [ "kneaddata" ];
```
(import check optional)